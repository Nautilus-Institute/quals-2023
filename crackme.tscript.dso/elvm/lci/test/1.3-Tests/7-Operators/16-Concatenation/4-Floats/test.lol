HAI 1.3
	VISIBLE SMOOSH "a" AN 1.234 AN "b" AN 2.345 AN "c" AN 3.456 AN "d" MKAY
KTHXBYE
