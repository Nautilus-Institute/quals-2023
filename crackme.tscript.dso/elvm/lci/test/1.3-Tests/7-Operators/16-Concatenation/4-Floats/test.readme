This test makes sure the concatenation operator works correctly on floating
point decimal arguments.
