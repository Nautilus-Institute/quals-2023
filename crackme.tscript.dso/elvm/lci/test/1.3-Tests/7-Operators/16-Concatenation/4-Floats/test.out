a1.23b2.34c3.45d
